src/skillmodels/check_model.py

```diff
@@ -15,12 +15,11 @@ def check_model(model_dict, labels, dimensions, anchoring):
         model_dict (dict): The model specification. See: :ref:`model_specs`
         dimensions (dict): Dimensional information like n_states, n_periods, n_controls,
             n_mixtures. See :ref:`dimensions`.
-
         labels (dict): Dict of lists with labels for the model quantities like
             factors, periods, controls, stagemap and stages. See :ref:`labels`
-
         anchoring (dict): Dictionary with information about anchoring.
             See :ref:`anchoring`
+        has_investments (bool): Whether the model has any investment factors
```

Comment:
I think there is a big gain from calling everything that is measured with error a latent factor. That is why in skillmodels, investment factors always have been just regular factors. I would call them **endogenous factors** now, but I would not use the term investments (and definitely not the abbreviation `inv`) to distinguish endogenous from exogenous factors.

Reply:
```suggestion
        has_concurrent_factors (bool): Whether the model has any factors that are influenced by others at the same point in time
```

Maybe that fits it best?